server/py/services/api/daemon.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import framework.service
+import services.api.main
+
+
+class Daemon(framework.service.Daemon):
+    def __init__(self, service_cls: framework.service.Service.__class__):
+        self._service = service_cls()
+
+    def initialize(self):
+        self._service.initialize()
+
+    @property
+    def service(self) -> services.api.main.Service:
+        return self._service
```

Comment:
@alonmr Post-merge question: Why duplicate the same methods from the `framework.service.Daemon` class? I know it is an `ABC` but it seems redundant...